front/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -15,6 +22,36 @@ const Row = styled.div`
   align-items: center;
 `;
 
+export const ADMIN_SITE_VIEW_FRAGMENT = gql`
```

Comment:
@rodctech moved the AdminSiteQuery Up a level so I could request the hideDonation within the same query and pass it down.

Reply:
Makes sense so we don't have to another query just for the hideDonation value